q2_gneiss/plot/_plot.py

```diff
@@ -124,7 +126,13 @@ def f(x):
             )
             sample_palette = pd.Series(sns.color_palette("Set2", 2),
                                        index=dcat.value_counts().index)
+
         elif isinstance(metadata, qiime2.CategoricalMetadataColumn):
+            if threshold is not None:
```

Comment:
When I updated the plugin to use the new Metadata API, I moved this check earlier in the method (see [this line](https://github.com/qiime2/q2-gneiss/pull/29/files#diff-9df90a58a11c42324dad341ada319215R40)). Removing the check here will probably fix the drop in coverage.